pyvista/utilities/helpers.py

```diff
@@ -1306,15 +1306,15 @@ class ProgressMonitor:
     algorithm
         VTK algorithm or filter.
 
-    message : str, optional
+    message : str, default: ""
         Message to display in the progress bar.
 
     scaling : float, optional
         Unused keyword argument.
 
     """
 
-    def __init__(self, algorithm, message="", scaling=100):
+    def __init__(self, algorithm, message="", scaling=None):
```

Comment:
If I'm reading things correctly, `scaling` isn't even used in this function. Why the change then, and can we just dump it? Or is this here for backwards compat?

Reply:
Ah, it's there in the docstring. Sorry. I understand then. Perhaps we should remove it eventually...

Reply:
It's correct that it's not used. I'm assuming that it's been kept here for backwards compatibility, so probably the right thing to do is to warn then raise during a later release.

It's a bit out of the scope of this PR and I should have left it alone.